train-simulation/simulator/config/node-red/entrypoint.sh

```diff
@@ -1,8 +1,20 @@
 #!/bin/sh
 
+# Commit of node-red.trainsim repository
+TRAINSIM_COMMIT="b4a68f144e33fba3b053ff92503e352426f2fdc8"
+REPOSITORY="node-red.trainsim"
+
 DIR=`pwd`
 
 cd /data
+
+wget "https://github.com/edgefarm/${REPOSITORY}/archive/${TRAINSIM_COMMIT}.zip"
+unzip ${TRAINSIM_COMMIT}.zip
+cp ${REPOSITORY}-${TRAINSIM_COMMIT}/* .
```

Comment:
Can't we unpack it directly into the destination=

Reply:
Ok.